code/src/java/pcgen/core/display/CharacterDisplay.java

```diff
@@ -1116,12 +1101,12 @@ public double getStatBonusTo(String aType, String aName)
 		return statBonusFacet.getStatBonusTo(id, aType, aName);
 	}
 
-	public String getUDamForRace()
+	String getUDamForRace()
```

Comment:
This is a negative change, IMHO.  The method itself can be used publicly, reducing the visibility (1) sends the wrong message because it implies there is some reason it needs package protection and (2) causes someone to have to change it back to public if it is needed publicly.  Semantics matter, and this is one of those areas where I think the automated tools that say "X could be Y" don't understand "Y" is a signaling mechanism and overusing it detracts from the code base.  The general purpose use of methods should be public, private, protected (when a subclass needs it in a template).  Package should be for things where it really is exposing internal behavior to the package for the purposes of keeping separate functions in different classes (e.g. avoiding inner classes)

Reply:
If this was a library I'd agree with you. Outside of library code all methods should be private by default and have minimal visibility outside of that. IMHO it is a language-mistake for package to be the default with no keyword (it should be `private`)